blueflood-core/src/test/java/com/rackspacecloud/blueflood/io/IOContainerTest.java

```diff
@@ -33,7 +33,7 @@
         "com.rackspacecloud.blueflood.utils.Metrics",
         "com.codahale.metrics.*"})
 @PrepareForTest({Configuration.class})
-@SuppressStaticInitializationFor( "com.rackspacecloud.blueflood.io.datastax.DatastaxIO")
+@SuppressStaticInitializationFor( { "com.rackspacecloud.blueflood.io.datastax.DatastaxIO", "com.rackspacecloud.blueflood.io.CassandraModel" })
```

Comment:
I was getting static initializing error for the CassandModel.QUOTED_KEYSPACE call in DatastaxLocatorIO.  But not having to use that anymore makes this no longer necessary.  Removed!  Thanks!